lib/dockerize_rails/docker_commands/docker_delete.rb

```diff
@@ -0,0 +1,37 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerDelete
+      def self.delete_rails
+        docker_delete DockerHelpers.get_name(:rails, :container)
+      end
+
+      def self.delete_mysql
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          return docker_delete DockerHelpers.get_name(:mysql, :container)
+        end
+        0
+      end
+
+      def self.delete_postgres
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          return docker_delete DockerHelpers.get_name(:postgres, :container)
+        end
+        0
+      end
+
+      def self.docker_delete(container_name)
+        container = Docker::Container.get(container_name)
+        container.stop if container.info['State']['Running']
+        container.delete(:force => true)
```

Comment:
__[rubocop]__
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

<!-- Comment by [SideCI](https://sideci.com) -->